src/manifest.json

```diff
@@ -25,22 +25,16 @@
   "content_scripts": [
     {
       "run_at": "document_end",
-      "matches": [
-        "https://www.faceit.com/*"
-      ],
-      "js": [
-        "content.js"
-      ]
+      "matches": ["https://www.faceit.com/*"],
+      "js": ["content.js"]
     }
   ],
   "browser_action": {
     "default_icon": "icon48.png",
     "default_popup": "popup.html"
   },
   "background": {
-    "scripts": [
-      "background.js"
-    ],
```

Comment:
Could you please revert these changes? Thanks

Reply:
Seems like `yarn prettier` formats also json files. My dev environment could also be misconfigured.